src/db.c

```diff
@@ -1210,6 +1211,13 @@ void scanGenericCommand(client *c, robj *o, unsigned long long cursor) {
                 return; */
             }
             i+= 2;
+        } else if (!strcasecmp(c->argv[i]->ptr, "novalues")) {
+            if (!o || o->type != OBJ_HASH) {
+                addReplyError(c, "Option 'novalues' can only be used in HSCAN command");
```

Comment:
Looking for similar error messages using `git grep -E "ReplyError.*option"` suggests something like this:

```suggestion
                addReplyError(c, "NOVALUES option can only be used in HSCAN");
```